libc/src/math/generic/sin.cpp

```diff
@@ -10,11 +10,9 @@
 #include "hdr/errno_macros.h"
 #include "src/__support/FPUtil/FEnvImpl.h"
 #include "src/__support/FPUtil/FPBits.h"
-#include "src/__support/FPUtil/PolyEval.h"
 #include "src/__support/FPUtil/double_double.h"
 #include "src/__support/FPUtil/dyadic_float.h"
 #include "src/__support/FPUtil/multiply_add.h"
-#include "src/__support/FPUtil/nearest_integer.h"
```

Comment:
Should sin.cpp now be including `src/math/generic/range_reduction_double_common.h` for `SIN_K_PI_OVER_128_F128` since  you moved it there?

Reply:
It is included in line 48, after we already include either `range_reduction_double_fma.h` or `range_reduction_nofma.h`, because the implementations in `range_reduction_double_common.h` needs `ONE_TWENTY_EIGHT_OVER_PI` selected from either of them.